xbmc/android/jni/AudioManager.cpp

```diff
@@ -80,6 +80,20 @@ int CJNIAudioManager::abandonAudioFocus(const CJNIAudioManagerAudioFocusChangeLi
                           "(Landroid/media/AudioManager$OnAudioFocusChangeListener;)I", listener.get_raw());
 }
 
+bool CJNIAudioManager::isBluetoothA2dpOn()
+{
+  return call_method<jboolean>(m_object,
+                          "isBluetoothA2dpOn",
```

Comment:
Odd indentation here (same in `isWiredHeadsetOn()`. Shouldn't it be moved to the same column as the `m_object` one line above?